app/src/main/res/values/strings.xml

```diff
@@ -117,24 +117,27 @@
         like better. With enough comparison picks, the app will create a ranking from whom you like
         the most to least.\n
         \n
-        Next the app limits every user to maximum 10 matches. You get matched based on whom you
+        The app limits every user to maximum 10 matches. You get matched based on whom you
         like the most and who likes you back. If you both are super popular, chances are you\'ll get
-        matched up unless either of you runs out of matches.\n
+        matched up unless either of you runs out of matches. If you are more "average" attractive, chances 
```

Comment:
The "average attractive" part is true. It's a bad message. We worked hard on that messaging with the FAQ to replace it with the idea of "local peaks" or more conversational people who are attracted to each other's niche.

Reply:
This one is tricky. I've reworded it a bit, but I'll need to come back to it and flesh it out.